app/helpers/merge_requests_helper.rb

```diff
@@ -20,6 +20,15 @@ def merge_request_pending_since mr
     "pending for #{time}"
   end
 
+  def merge_request_counter (value)
```

Comment:
@hugopl, ok I will do that. Actually I installed the rubocop gem in my development environment but I used "()" to improve the readability but I just realized that you weren't using it after the push.